.github/workflows/notify-schema-change.yml

```diff
@@ -0,0 +1,57 @@
+name: Notify Analytics Config Changes
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'config/analytics_hidden_pii.yml'
+      - 'config/analytics_blocklist.yml'
+      - 'config/analytics.yml'
+      - 'config/analytics_pii.yml'
+  workflow_dispatch:
+
+jobs:
+  notify-slack:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Get analytics config changes
+        id: analytics-diff
+        run: |
+          echo "ANALYTICS_DIFF<<EOF" >> $GITHUB_ENV
+          git diff HEAD~1 HEAD -- config/analytics_hidden_pii.yml config/analytics_blocklist.yml config/analytics.yml config/analytics_pii.yml >> $GITHUB_ENV
```

Comment:
```
git diff HEAD~1 HEAD
```
not sure this is going to work? if the branch is over several commits and the change is not in the last one don't think it'll pick up on the change?

Reply:
Good spot! Have updated it to compare against the state of master before the merge